DOC-377 Add k8s md files (#1160)
RedisLabs/redislabs-docs: content/platforms/kubernetes/reference/redis_enterprise_cluster_api.md

---

> Title: Redis Enterprise Cluster API

Reviewer: ```suggestion
Title: Redis Enterprise Cluster - Custom Resource Specification
```

Reviewer 2: This document is a more complete version of "cluster_options.md". We should not have both.

Reviewer 2: This is a more complete version of cluster-options.md and we should not include both.

description: 
weight: 70
alwaysopen: false
categories: ["Platforms"]
aliases:
---

This document describes the parameters for the Redis Enterprise Cluster custom resource.

> > Note this document is auto-generated from code comments. To contribute a change please change the code comments.

Reviewer: ```suggestion
> Note this document is auto-generated from the Operator code.
```
This isn't actionable for the docs audience since the repo where you can contribute these changes is closed source. Any contributions here will have to be addressed in that repo.
